.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on:
+  merge_group:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  test:
+    name: Test
+    runs-on: macos-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Install cairo
+        run: |
+          curl -L https://raw.githubusercontent.com/lambdaclass/cairo-by-example/main/build/installer.sh | bash
+
+      - name: Use node
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+
+      - name: Install txm
+        run: npm install -g txm
+
+      - name: Test with txm
+        run: | 
+          export CARGO_MANIFEST_DIR="/usr/local/Cellar/cairo-lang/v2.0.0-rc4/corelib"
```

Comment:
I think we should use `v2.1.0-rc1`. It adds a flag `--single-file`, needed when running from a single file. That way we don't need to update the test commands in each example later.

Reply:
Sorry I didn't quite catch that, could you elaborate? The commands to run need to be specified in every .md

Reply:
Nevermind. I didn't see we are using the install script. I was just saying that when we update to the latest compiler version, we'll need to add a `--single-file` flag to every `cairo-run` command.